src/frontend/next/src/components/Banner.tsx

```diff
@@ -1,5 +1,143 @@
-import { FC } from 'react';
+import { useEffect, useState } from 'react';
+import { makeStyles } from '@material-ui/core/styles';
```

Comment:
change this line to: `import { makeStyles, Theme, createStyles, Fab, Grid, Typography } from '@material-ui/core';`